keylime/keylime_conf_files/keylime.conf.edgeaiApp

```diff
@@ -342,7 +342,7 @@ ek_check_script=
 [registrar]
 #=============================================================================
 
-# The registrar's IP address and port used to communicate with other 
+# The registrar's IP address and port used to communicate with other
 # services as well as a bind address for the registrar server.
 registrar_ip = 127.0.0.1
```

Comment:
This will change, In the current setup I have a 3 node cluster, with two nodes running containerized agents, and another running a containerized registrar, eventually I would like the have the registrar running on the same machine as the verifier, i.e the external VM

Reply:
No problem, I fully expect that value to change. Have not got round to setting anything specific, as that will depend on the actual environment we use and whatever CIDR it has.

Reply:
Sounds good!